packages/datatrak-web/src/Routes.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import { Route, Routes as RouterRoutes } from 'react-router-dom';
+import { LandingPage } from './views';
+
+/**
+ * This Router is using [version 6.3]{@link https://reactrouter.com/en/v6.3.0}, as later versions are not supported by our TS setup. See [this issue here]{@link https://github.com/remix-run/react-router/discussions/8364}
+ * This means the newer 'createBrowserRouter' and 'RouterProvider' can't be used here.
+ *
+ * **/
+
+export const Routes = () => {
+  return (
+    <>
```

Comment:
```suggestion
  return (

```
Don't need the fragment